src/OpenApi/Factory/OpenApiFactory.php

```diff
@@ -385,24 +383,16 @@ private function collectPaths(ApiResource $resource, ResourceMetadataCollection
                         $successStatus = (string) $operation->getStatus() ?: 201;
                         $openapiOperation = $this->buildOpenApiResponse($existingResponses, $successStatus, \sprintf('%s resource created', $resourceShortName), $openapiOperation, $operation, $responseMimeTypes, $operationOutputSchemas, $resourceMetadataCollection);
 
-                        if (null === $errors) {
-                            $openapiOperation = $this->addOperationErrors($openapiOperation, [
-                                $defaultError->withStatus(400)->withDescription('Invalid input'),
-                                $defaultValidationError,
-                            ], $resourceMetadataCollection, $schema, $schemas, $operation);
-                        }
```

Comment:
The idea behind this is that if an error is defined by the user we do not add our custom errors.